cli/pkg/edgeclient/client.go

```diff
@@ -74,20 +82,18 @@ func (c *Client) AddNewModelWithImage(
 	modelFormatName := "ContainerImage"
 	// This will be used to flag the model as edge compatible (i.e. has the required metadata to be built by the edge pipeline)
 	parameters["edgeCompatible"] = "true"
-	externalID := getImageID(modelName, modelVersion, modelName)
 	md, err := modelregistry.ToMetadataValueMap(parameters)
 	if err != nil {
 		return nil, fmt.Errorf("failed to add model image: %w", err)
 	}
 	m, v, a, err := c.modelRegistryClient.AutoRegisterModelVersionArtifact(
-		modelName, modelDescription, modelVersion, modelName, externalID, uri, modelFormatName, "", md,
+		modelName, modelDescription, modelVersion, modelName, uri, modelFormatName, "", md,
 	)
 	if err != nil {
 		return nil, fmt.Errorf("failed to add model image: %w", err)
 	}
 
 	return &ModelImage{
-		ID:          a.ModelArtifact.GetExternalID(),
 		ModelID:     m.GetId(),
```

Comment:
The reason I used this initially was to be able to look up an artifact by a globally unique ID as that doesn't exist  in MR. But since I've moved the UX to just identify an artifact by the `--model-id <id> --version <version>`  , then that globally unique ID is no longer needed.